tests/main.test.ts

```diff
@@ -565,7 +565,7 @@ function createContext(
       reviewDelayTolerance: "3 Days",
       taskStaleTimeoutDuration: "30 Days",
       maxConcurrentTasks: 3,
-      startRequiresWallet,
+      startRequiresWallet: false,
```

Comment:
Why was this changed?

Reply:
while rebasing it seems like i merged a new commit from development branch that includes that change, i’ll revert and commit new changes soon.